### PR TITLE
Bugfix/498 fix package version

### PR DESCRIPTION
**Pull request by davidzwa**
- Cleanup npm_package_version calls, make it less dependent
- Fix startup sync up version
- Move nodemon to dev-dep
- Log your changes!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "octofarm",
-  "version": "1.1.13",
+  "version": "1.1.15",
```
Comment (NotExpectedYet) [inline package.json:3]: 1.1.15 - we're on may right? xD

File: server_src/runners/softwareUpdateChecker.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
 const Logger = require("../lib/logger.js");
-const { getGithubReleasesPromise } = require("./githubClient");
+const {AppConstants} = require("../app.constants");
+const {getGithubReleasesPromise} = require("./githubClient");
 
-const logger = new Logger("OctoFarm-SoftwareUpdateChecker");
+const logger = new Logger("OF-SoftwareUpdateChecker");
```
Comment (NotExpectedYet) [inline server_src/runners/softwareUpdateChecker.js:5]: Minor here but I think "OctoFarm" was fine initially unless we're changing them all to match?

**Review by NotExpectedYet — CHANGES_REQUESTED** [review]

File: server_src/routes/index.js [hunk 3]
```diff
@@ -26,8 +26,7 @@ const { fetchMongoDBConnectionString } = require("../../app-env");
 const { isPm2, isNodemon, isNode } = require("../utils/env.utils.js");
 
 const SystemInfo = systemInfo.SystemRunner;
-const version = process.env.npm_package_version;
-console.log(`Version: ${version} (server started)`);
```
Comment (davidzwa) [inline server_src/routes/index.js:30]: I dont index.js should produce any logs at startup as it is run soooo many times in tests.

Comment (NotExpectedYet) [reply server_src/routes/index.js:30]: No worries, was gonna remove that myself it's legacy now anyway

**Review by NotExpectedYet — APPROVED** [review]

**Comment by mkevenaar** [comment]
I can confirm that this works!

**Comment by davidzwa** [comment]
Need changelog still

File: CHANGELOG.md [hunk 4]
```diff
@@ -43,7 +43,8 @@ All notable changes to this project will be documented in this file.
     - Fixed #468: Disable update notification and buttons to docker installs. 
     - Fixed #452: Docker documnetation was missing path for /images.
     - Fixed #478: Abort with a friendly message if Node version less than 14 is detected.
-    - Fixed #429 and #378: Memory/CPU graphs in system page now tolerant to missing values so it can show no matter what. 
+    - Fixed #429 and #378: Memory/CPU graphs in system page now tolerant to missing values so it can show no matter what.
+    - Fixed #498: Fix package version not always preset and synced correctly when not running npm commands, f.e. pm2.
```
Comment (NotExpectedYet) [inline CHANGELOG.md:47]: #500 

This is my fault but I did it like this.

**Review by NotExpectedYet — CHANGES_REQUESTED** [review]

**Review by NotExpectedYet — APPROVED** [review]

**Pull request merged**